### ISSUE TITLE
[log][v2] There is no information on how to change the log levels for each target

### ISSUE THREAD

**Issue by ktecho** [tauri-apps/plugins-workspace#1642]
I've setup the log exactly how it's specified in the documentation, but the file and stdout are getting flooded with TRACE logs. How can I change the level of the logs that go there?

**Comment by SteveLauC** [comment]
Another bump! I would like to have all the logs written to the log file. But for stdout/stderr, I only want the logs that I am interested in :D

**Comment by JPHutchins** [comment]
Yes, I've been struggling with this and it appears to be a bug or a missing feature. Not really sure how to proceed.

**Comment by SteveLauC** [comment]
I eventually implemented dynamic log level support similar to what is offered by the [`env_logger`](https://docs.rs/env_logger/latest/env_logger/#example) crate. Then, in development mode, I set the log level manually, picking up the logs I am interested in. For the built apps, I will set the log level to "full" so that I can capture all of them in the log file.

https://github.com/infinilabs/coco-app/blob/24b0123a614ff3b86a2945cc274ed45d4319e3fa/src-tauri/src/lib.rs#L456-L586
